### PR TITLE
Add feature flagging. Cache cli version

### PR DESCRIPTION
**Pull request by leeweisberger**
## Description
This pr contains two changes
1. Add feature flagging capability. Cache feature flags in the airplane config file for an hour. Run the bundle deploy command if the corresponding feature flag is on
2. Cache CLI version in the airplane config file rather than checking for new versions of the cli on each execution for one day.

## Test plan
I manually tested both of these

### CODE REVIEW

File: cmd/airplane/root/deploy/deployer_test.go [hunk 1]
```diff
@@ -233,3 +232,52 @@ func TestDeploy(t *testing.T) {
 		})
 	}
 }
+
+func TestParseRemote(t *testing.T) {
```
Comment (leeweisberger) [inline cmd/airplane/root/deploy/deployer_test.go:236]: Moved from tasks/deploy

File: cmd/airplane/tasks/deploy/deploy.go [hunk 2]
```diff
@@ -99,14 +103,23 @@ type taskDeployedProps struct {
 }
 
 func run(ctx context.Context, cfg config) error {
+	l := logger.NewStdErrLogger(logger.StdErrLoggerOpts{WithLoader: true})
+	defer l.StopLoader()
+
+	if cfg.root.Flagger.Bool(ctx, l, flagsiface.DeployBundles) {
```
Comment (leeweisberger) [inline cmd/airplane/tasks/deploy/deploy.go:109]: If the flag is on, run bundle deploy

File: pkg/flags/flagsiface/flags.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package flagsiface
```
Comment (leeweisberger) [inline pkg/flags/flagsiface/flags.go:1]: copied this interface from airport

Comment (ameyakhare) [reply pkg/flags/flagsiface/flags.go:1]: should be in lib then, no?

**Review by ameyakhare — APPROVED** [review]
seems fine

File: pkg/flags/flagsiface/flags.go (hunk 3 above)
Comment (leeweisberger) [reply pkg/flags/flagsiface/flags.go:1]: It should be :/

**Pull request merged**